content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,582 @@ import Tooltip from "shared/components/ui/Tooltip";
+- Patch versions are provided within four weeks of a new Kubernetes release.
+
+In the period of time following a Kubernetes release, the Kubernetes pack undergoes rigorous vulnerability scans and penetration testing. 
+
+A minor Kubernetes version will be deprecated in Palette when the Kubernetes community announces the version is entering End of Life (EOL). The deprecated version will remain in that state for three months and will be disabled after a quarter and deleted from Palette in the following quarter.
+
+<br />
 
 <WarningBox>
-Info: Only Kubernetes versions 1.19 and above is supported
-</WarningBox>
 
-## Supported Kubernetes versions
+When updating to a new Kubernetes version, these rules apply:
+
+<br />
 
-<InfoBox>
+- You cannot copy the pack configuration from one version to another.
 
-Versions supported in the latest [release](/release-notes/) are highlighted.
 
-</InfoBox>
+- Updating to a newer Kubernetes version must be done incrementally, one minor version at a time. 
+
+</WarningBox>
+
+
+# Versions Supported
+
 
 <Tabs>
 
-<Tabs.TabPane tab="1.19.x" key="k8s_v1.19">
-
-- **1.19.16**
-- **1.19.15**
-- **1.19.14**
-- **1.19.13**
-- **1.19.12**
-- **1.19.11**
-- **1.19.10**
-- **1.19.9**
-- **1.19.8**
-- **1.19.7**
-- **1.19.6**
-- **1.19.5**
-- **1.19.4**
-- **1.19.3**
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
 
-</Tabs.TabPane>
+## Prerequisites
 
-<Tabs.TabPane tab="1.20.x" key="k8s_v1.20">
-
-- **1.20.12**
-- **1.20.11**
-- **1.20.10**
-- **1.20.9**
-- **1.20.8**
-- **1.20.7**
-- **1.20.6**
-- **1.20.5**
-- **1.20.4**
-- **1.20.2**
-- **1.20.1**
-- **1.20.0**
+- A minimum of 4 CPU and 4GB Memory.
 
-</Tabs.TabPane>
+- Operating System (OS) dependencies as listed in the table.
 
-<Tabs.TabPane tab="1.21.x" key="k8s_v1.21">
+| OS Distribution | OS Version | Supports Kubernetes 1.26.x |
+|---------------|------------|----------------------------|
+| CentOS        | 7.7        | ❌                         |
+| Ubuntu        | 22.04      | ✅                         |
+| Ubuntu        | 20.04      | ❌                         |
+| Ubuntu        | 18.04      | ❌                         |
 
-- **1.21.10**
-- **1.21.6**
-- **1.21.5**
-- **1.21.4**
-- **1.21.3**
-- **1.21.2**
-- **1.21.1**
-- **1.21.0**
 
-</Tabs.TabPane>
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| ``pack:k8sHardening``  | Flag to decide if Kubernetes hardening should be applied. Default: ``True``. When set to ``True``, additional flags configured in `kubeadmconfig` will be honored and will be set to the corresponding components.|
+| ``pack:podCIDR`` | The CIDR range for Pods in cluster. This should match the networking layer property ``calicoNetworkCIDR``. Default: `192.168.0.0/16`|
+| ``pack:serviceClusterIpRange`` | The CIDR range for services in the cluster. This should not overlap with any IP ranges assigned to nodes for pods. Default: `10.96.0.0/12`|
```

Comment:
There's a typo in here. Should be "nodes or pods"

Reply:
Good catch @brathina-spectro. Fixed typo throughout doc.